.travis.yml

```diff
@@ -1,39 +1,32 @@
----
 language: node_js
 
 sudo: false
 
+matrix:
+  include:
+    - node_js: 0.12
```

Comment:
Wouldn't it make sense to use the latest version of node with the other scripts?